Update README.md (#2)
chrisngyn/LCwithJustin: README.md

# LCwithJustin

>  A group of computer science noobs just doing mock interviews with each other

Reviewer: ```suggestion
Justin is my friend and we do LeetCode + mock interviews with each other. 👍 
```
